src/services/AuthenticationUserService.js

```diff
@@ -1,19 +1,41 @@
 const bcrypt = require("bcrypt");
+const Response = require("../utils/responses");
 const User = require("../models/User");
 
-const jwtGenerate = require("../utils/jwtGenerate");
+const JwtManager = require("../utils/jwtManager");
 
 module.exports = async (req, res) => {
   const { email, password } = req.body;
 
   const user = await User.findOne({ email }).select("+password");
 
-  if (!user) return res.status(404).json({ error: "User not found" });
+  const jwt = new JwtManager();
+
+  const response = new Response(res);
+  const { entities } = response;
+
+  if (!user)
+    return response
+      .isError()
+      .entity(entities.USER)
+      .code(response.RESOURCE_NOT_FOUND)
+      .message("User not found")
+      .send();
 
   if (!password || !(await bcrypt.compare(password, user.password)))
-    return res.status(400).json({ error: "Invalid Password" });
+    return response
+      .isError()
+      .entity(entities.USER)
+      .code(response.INVALID_REQUEST)
+      .message("Invalid Password")
+      .send();
 
   user.password = undefined;
 
-  return res.status(200).json({ user, token: jwtGenerate({ id: user.id }) });
+  return response
+    .entity(entities.USER)
+    .code(response.SUCCESS_GET)
```

Comment:
It was strange to use SUCCESS GET in a post request? You might want to find a more generic name for SUCCESS_GET.

Using the names of HTTP statuses, would that be a good one? What do you think about this?